cumulus_library/template_sql/statistics/count.sql.jinja

```diff
@@ -4,6 +4,8 @@
 {%- endif -%}
 {%- endmacro -%}
 
+{#- TODO: move secondary/tertiary resource declaration upstream to
```

Comment:
these are my attempt to address the dangling comment from the prior PR i pulled the trigger on early

Reply:
You didn’t pull early really, I was just being verbose in the comments 😄